### DISCUSSION TITLE
Cap on number of participants on a given date

### DISCUSSION THREAD

**Discussion by dustyr** [lukevella/rallly#155] [Ideas]
Would like to use Rallly to schedule online classes.  I want people to sign up for dates but I want to cap the number of participants on any given date.  As an example, if I offer a class on Monday, Wednesday and Friday, I only want up to 20 people to be able to say yes to any one of the dates on a first-come, first serve basis.  After that, It would not allow additional participants. From what I can tell this is not currently available so offering it as an enhancement idea.  

THANK YOU FOR ALL YOUR EFFORT ON THIS!!!!

**Comment by lukevella** [comment]
This is definitely a popular request and it's in the backlog so you can expect to see it in the near future.

**Reply by ZenPadavan** [reply]
Hoi Luke,
I would like to participate to develop this feature, since I need it...
Can you guide me, how to start to implement it?
Should I fork this project and develop it for myself, or can we do it together into the main product with your guidance?

**Reply by taker93** [reply]
I added this feature in a fork last year: https://github.com/taker93/rallly

But did not have the quality for a pull request. ;-)

**Comment by justlonger** [comment]
Is this feature available now?

**Comment by TriangularCube** [comment]
Yeah it's been 2 years since _near future_ was mentioned, any movement on this?
